proj/src/main/webapp/chat.js

```diff
@@ -887,13 +895,15 @@ function addUserInfoToDom(userObj) {
     tagContainer.innerHTML = '';
     
     if (userObj.uid !== currentUid) {
-            friendRequestButton(userObj.uid);
-            blockButton(userObj.uid);
+        friendRequestButton(userObj.uid);
+        blockButton(userObj.uid);
+        document.getElementById('change-pfp').classList.add('hidden');
```

Comment:
Just like a couple lines below, can you use the `.hidden=true;` instead of using a class.